droidmediacodec.cpp

```diff
@@ -457,6 +457,10 @@ class DroidMediaCodecBuilder {
         format->setInt32("profile", OMX_VIDEO_AVCProfileBaseline);
```

Comment:
There are probably other hardcoded values here that you could add to the new codec-specific settings struct, such as profile and level. Do you need control of any more for this?

Reply:
Yes, that would be better, thanks.